Assets/09_Data/01_Levels/LVL_Level 08.asset

```diff
@@ -0,0 +1,58 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 4112a7d61daa462a8db5564fcc6092d5, type: 3}
+  m_Name: LVL_Level 08
+  m_EditorClassIdentifier: 
+  _levelNumber: 8
+  _closeAfterMinutes: 2
+  _closeAfterSeconds: 0
+  _targetSpecies: {fileID: 11400000, guid: 24d1da95cc68a5345b360b8a6519932a, type: 2}
+  _targetPosition: 5
+  _amountOfSpeciesInLevel: 9
+  _targetExists: 1
+  _requiredScoreStar1: 2100
+  _requiredScoreStar2: 2300
+  _requiredScoreStar3: 2450
+  _star1Percentage: 82.67716
+  _star2Percentage: 90.55118
+  _star3Percentage: 96.456696
```

Comment:
I hope this is aimed to be at `94.4%`. Please verify.

Reply:
Why is that an issue? 

In any case it is meant to be super difficult to get 3 starts in this level, if not nearly impossible, just to demonstrate the pain points before the monetization opportunities (future feature "upgrades")

Reply:
No issue. Just verification.